packages/lib-classifier/src/plugins/drawingTools/components/Point/Point.js

```diff
@@ -22,7 +22,7 @@ function Point ({ active, children, mark, scale }) {
   const radius = RADIUS[size] / scale
 
   return (
-    <g transform={`translate(${mark.x}, ${mark.y})`}>
+    <g>
```

Comment:
Why was this removed?

Reply:
It wasn't. It's moved to `DrawingToolRoot`.